environment/environment_templates.go

```diff
@@ -292,7 +296,7 @@ func NewChainlinkCluster(nodeCount int) K8sEnvSpecInit {
 	dependencyGroup := getBasicDependencyGroup()
 	addPostgresDbsToDependencyGroup(dependencyGroup, nodeCount)
 	dependencyGroups := []*K8sManifestGroup{dependencyGroup}
-	return addNetworkManifestToDependencyGroup("basic-chainlink", chainlinkGroup, dependencyGroups)
```

Comment:
With the removal of all these envNames like basic-chainlink, is there an easy way to determine what the namespace is now? Does a multi-network deploy create multiple namespaces?